src/backend/distributed/planner/multi_explain.c

```diff
@@ -1461,7 +1461,7 @@ WrapQueryForExplainAnalyze(const char *queryString, TupleDesc tupleDesc)
 static List *
 SplitString(const char *str, char delimiter)
 {
-	size_t len = strnlen_s(str, RSIZE_MAX_STR);
+	size_t len = strlen(str);
```

Comment:
Here I used `strnlen(str, RSIZE_MAX_MEM)` I guess there isn't a big difference here but using `strnlen` might be a bit safer maybe?

https://github.com/citusdata/citus/pull/4170/files#diff-44a470ebd7842955b493ca913941c03dR1464

Reply:
either way works for me, though given that this is a utility function I'll make it a parameter